lf-to-abstract-sql.ometajs

```diff
@@ -214,7 +214,7 @@ export ometa LF2AbstractSQL <: SBVRCompilerLibs {
 		AddRelationship(baseTable, factType.slice(1), fkName, references)
 		GetResourceName(factType):factTypeResourceName
 		{
-			_.each($elf.synonymousForms[factTypeResourceName], function(synForm) {
+			_.forEach($elf.synonymousForms[factTypeResourceName], function(synForm) {
```

Comment:
Did you mean

```suggestion
			$elf.synonymousForms[factTypeResourceName].forEach(function(synForm) {
```

?

Reply:
Nope, this is an object - I included it as standardizing on forEach to match the native version